content/en/monitors/guide/custom_schedules.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Customize monitor evaluation frequencies
+kind: Guide
+disable_toc: false
+further_reading:
+- link: "https://docs.datadoghq.com/monitors/configuration/?tab=thresholdalert#evaluation-frequency"
+  tag: "Documentation"
+  text: "Learn about monitor evaluation frequency"
+- link: "/monitors/downtimes"
+  tag: "Documentation"
+  text: "Downtimes"
+- link: "/monitors/configuration/?tab=thresholdalert#evaluation-window"
+  tag: "Documentation"
+  text: "Cumulative time windows"
+---
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Monitor custom schedule is in private beta. To request access, contact Support at support@datadoghq.com.
```

Comment:
Is email the best way? I've most often seen links to the help page (https://docs.datadoghq.com/help/) when we ask people to contact Support.